server/base/src/main/java/org/apache/accumulo/server/rpc/TServerUtils.java

```diff
@@ -231,8 +236,8 @@ public static ServerAddress startServer(MetricsSystem metricsSystem, ServerConte
    */
   public static ServerAddress createThreadedSelectorServer(HostAndPort address,
       TProcessor processor, TProtocolFactory protocolFactory, final String serverName,
-      final int numThreads, final int numSTThreads, long timeBetweenThreadChecks,
-      long maxMessageSize) throws TTransportException {
+      final int numThreads, final long threadTimeOut, final int numSTThreads,
```

Comment:
I was trying to ensure the parameter order was correct when looking at this.  Once I realized you had consistently put the timeout after the number of threads here and elsewhere it made verify the order much easier.

Reply:
cool